### PR TITLE
Javascript(ES6) Capstone Project

### PR DESCRIPTION
**Pull request by bushmusi**
> # We have added below features in this capstone project

1. We have used JavaScript to make websites dynamic and build single-page apps.
2. We have used basic *ES6* syntax and  Modules
3. We have used `callbacks` and `promises`
4. Unit `tests` for comment and item counter
5. Send and receive data from an API

> ## Usefull Info

1. We are only two coding partners only so we are not supposed to implement **reservation** feature
2. [ Project Requirement link.](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

### CODE REVIEW

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi AckonSamuel, 

Good job so far!

- [ ] Major highlights

- The PR summary and commit messages are very descriptive.
- The codebase is clean, easy to read and understand.
- Passed linters check

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Record a video presentation of the project and add it to the readme. It's part of the project requirement.

I had to add this comment here because the readme file is not part of the PR.
  
_Check the comments under the review._

## Optional suggestions

- [ ] The demo link in the readme takes me to a `todo list`. It will be nice to remove it or replace with the right link.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bushmusi** [comment]
> Hi AckonSamuel,
> 
> Good job so far!
> 
> * [ ]  Major highlights
> * The PR summary and commit messages are very descriptive.
> * The codebase is clean, easy to read and understand.
> * Passed linters check
> 
> However, there are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> * [ ]  Record a video presentation of the project and add it to the readme. It's part of the project requirement.
> 
> I had to add this comment here because the readme file is not part of the PR.
> 
> _Check the comments under the review._
> 
> ## Optional suggestions
> * [ ]  The demo link in the readme takes me to a `todo list`. It will be nice to remove it or replace with the right link.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Code review by **Eric Enaburekhan**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @enaburekhan 

Thanks for yourfeedback. You mention the right things. We are going to add it in a moment.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Todo List
-This is a book shelve website dedicated to store collection of books, add new books and delete books. 
+# Space Flight News
+This project is about consuming `REST` API, We have used ES6 modules and synatax, Linters, Github  and GitFlow, Test using JEST package. We also use Promise object for consuming REST API.
 
-In this project, you will be using javascript arrays and object to store the list of books and saving the book colection in the local storage. Do not use any other framework, just pure javascript. After you are done with the project, run tests and deploy.
+### [Live Demo Link](https://ackonsamuel.github.io/space-flight-news/)
+### [Video presentation](https://drive.google.com/file/d/1gs5j-7onBObNtN2H7lzD0URjXKFfELMQ/view?usp=sharing)
```
Comment (Whoistolu) [inline README.md:7]: - Kindly ensure that all the team members participate in the presentation and remember to also highlight a piece of code that each team member is proud of.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,7 @@
 This project is about consuming `REST` API, We have used ES6 modules and synatax, Linters, Github  and GitFlow, Test using JEST package. We also use Promise object for consuming REST API.
 
 ### [Live Demo Link](https://ackonsamuel.github.io/space-flight-news/)
-### [Video presentation](https://drive.google.com/file/d/1LeuCI0VjSKb6Y--YXMLOsuF_Z5PJnFKU/view?usp=sharing)
```
Comment (Whoistolu) [inline README.md]: - Kindly ensure that all the team members participate in the presentation and remember to also highlight a piece of code that each team member is proud of.

Comment (Whoistolu) [reply README.md]: Great job implementing the changes recommended 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @bushmusi  and @AckonSamuel 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/assets/js/setNews.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const setNumNews = (num) => {
+  const newsCounter = document.getElementById('news-counter');
+  newsCounter.innerText = num;
+  console.log(newsCounter.innerText);
```
[the remark this answers is not in the document]
Comment (bushmusi) [reply src/assets/js/setNews.js:4]: Noted, and fixed 👍

**Pull request merged**